### PR TITLE
[1363] Added the azure related settings for creds for backend

### PR DESCRIPTION
**Pull request by defong**
### Context
Appsetting key `SETTINGS__AUTHENTICATION__SECRET`

### Changes proposed in this pull request
Added the azure related settings for creds for backend

### Guidance to review
related to https://github.com/DFE-Digital/bat-infrastructure/pull/124

@OutKa5t @smneal fyi

### CODE REVIEW

**Comment by defong** [comment]
[Allow mc-support to grant access requests through mc-be](https://trello.com/c/VdmjSTgi/1363-allow-mc-support-to-grant-access-requests-through-mc-be)

File: azure/template.json [hunk 1]
```diff
@@ -267,6 +273,10 @@
                                 "name": "RAILS_SERVE_STATIC_FILES",
                                 "value": "[parameters('serveStaticFiles')]"
                             },
+                            {
+                                "name": "SETTINGS__AUTHENTICATION__SECRET",
+                                "value": "[parameters('settingsAuthenticationSecret')]"
```
Comment (misaka) [inline azure/template.json]: Is this secret being sent to `manage-courses-backend`? If so, what about naming it so that that's clear?

Comment (defong) [reply azure/template.json]: its stated on line 135

Comment (defong) [reply azure/template.json]: for consistence I keep it the same as it is on frontend

Comment (timabell) [reply azure/template.json]: https://github.com/DFE-Digital/manage-courses-frontend/blob/master/azure/template.json#L214

Comment (timabell) [reply azure/template.json]: https://github.com/DFE-Digital/bat-infrastructure/pull/124

Comment (misaka) [reply azure/template.json]: please please please can we call it `SETTINGS__MANAGE_BACKEND__SECRET`, just based off of the naming we currently use already in mc-fe?

Comment (defong) [reply azure/template.json]: https://trello.com/c/Riw3vaQe/1440-settingsauthenticationsecret-on-fe-and-support-needs-to-be-renamed

**Pull request merged**